pkg/distro/rhel/imagetype.go

```diff
@@ -284,6 +290,47 @@ func (t *ImageType) Manifest(bp *blueprint.Blueprint,
 		staticPackageSets[name] = getter(t)
 	}
 
+	var itSpec struct {
+		Spec struct {
+			Packages        []string `yaml:"packages"`
```

Comment:
Just to clarify: the current merge rules and their implementation merge lists just for top-level keys, so moving `Include` and `Exclude` under `Packages` would break everything. However, we can alter the merge rules:

Dictionaries:
    If both values are dictionaries, perform a recursive merge:
        For each key:
            If the key exists in both, merge the corresponding values.
            If the key exists in only one, use that value.

Lists:
    If both values are lists, merge them:
        Append items from the second list to the first.
        Deduplicate items, maintaining all unique entries.

Scalars (Strings, Numbers, Booleans, Null):
    If both values are scalars, the value from the second structure overrides the first.

Type Mismatch:
    If the types of values for a given key differ, the merge is not allowed. This results in an error.

Reply:
Aha, sorry, we can start with the exiting top-level logic, right now the format is internal and as long as we do not have external image defintions we can always refactor. So +1 with starting from that (even though I think we should get something better soon(ish)).